<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_editarprodutovenda</class>
 <widget class="QDialog" name="fm_editarprodutovenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Produto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Arquivo_de_recursos.qrc">
    <normaloff>:/imagens/img/gestao-de-dados.png</normaloff>:/imagens/img/gestao-de-dados.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_edita_produto"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Preço Unitário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_edita_precoUnitario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Quantidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_edita_quantidade"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_edita_confirma">
        <property name="text">
         <string>Confirmar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_edita_cancela">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Arquivo_de_recursos.qrc"/>
 </resources>
 <connections/>
</ui>
